docs/capabilities.md

```diff
@@ -95,3 +95,4 @@ title: Capabilities
 
 ## 15
 * `chat-permission` - When permission 128 is required to post chat messages, reaction or share items to the conversation
+* `config => call => enabled` - Whether calling is enabled on the instance or not
```

Comment:
should the capability be really named like this??? @nickvergessen

Reply:
The capabilities have an array structure. Since it's a config and can be changed by admins on the fly instead of being a version based feature flag, I put it into the config section:
```
        "spreed": {
          "features": [
            "audio",
            "video",
            "chat-v2",
            "conversation-v4",
            "guest-signaling",
            "empty-group-room",
            "guest-display-names",
            "multi-room-users",
            "favorites",
            "last-room-activity",
            "no-ping",
            "system-messages",
            "delete-messages",
            "mention-flag",
            "in-call-flags",
            "conversation-call-flags",
            "notification-levels",
            "invite-groups-and-mails",
            "locked-one-to-one-rooms",
            "read-only-rooms",
            "listable-rooms",
            "chat-read-marker",
            "chat-unread",
            "webinary-lobby",
            "start-call-flag",
            "chat-replies",
            "circles-support",
            "force-mute",
            "sip-support",
            "chat-read-status",
            "phonebook-search",
            "raise-hand",
            "room-description",
            "rich-object-sharing",
            "temp-user-avatar-api",
            "geo-location-sharing",
            "voice-message-sharing",
            "signaling-v3",
            "publishing-permissions",
            "clear-history",
            "direct-mention-flag",
            "notification-calls",
            "conversation-permissions",
            "rich-object-list-media",
            "rich-object-delete",
            "reactions"
          ],
          "config": {
            "attachments": {
              "allowed": true,
              "folder": "/Talk"
            },
            "call": {
              "enabled": true
            },
            "chat": {
              "max-length": 32000,
              "read-privacy": 0
            },
            "conversations": {
              "can-create": true
            },
            "previews": {
              "max-gif-size": 3145728
            }
          }
        },
```